manifest.webapp

```diff
@@ -13,6 +13,7 @@
   },
   "version": "1.85.0",
   "licence": "AGPL-3.0",
+  "offline_support": true,
```

Comment:
Is this documented somewhere? If not, we should do it

Reply:
The documentation is not merged yet, but you can preview it there: https://github.com/cozy/cozy-flagship-app/blob/28394cac703df96e00274cef6e985a8060046dd1/docs/how-to-make-cozy-app-offline-compatible.md

Reply:
Ok, we should also think to add it here: https://docs.cozy.io/en/cozy-apps-registry/#properties-meaning-reference